app/deed/search/routes.py

```diff
@@ -0,0 +1,7 @@
+from app.deed.search import views
+
+
+def register_routes(blueprint):
+    @blueprint.route('/deed/search')
```

Comment:
Can you limit the methods on this? I assume only GET is needed? `/search', methods=['GET']`